README.md

```diff
@@ -1,68 +1,24 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
 
-In the project directory, you can run:
 
-### `npm start`
+//*******************************************************//
```

Comment:
this is okay not existing OR you can use some Markdown to style things
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

you can use a horizontal rule here like this:

---

and also headers like this
## this is a header
### this is a less important header

**_formatting_**